esp_link/esp_link.py

```diff
@@ -115,16 +116,10 @@ def __init__(self, loop, verbose):
             await self.swriter.awrite(line.decode('utf8'))
             self.verbose and print('Sent', line, 'to Pyboard app\n')
 
-    async def server_status(self):  # TODO Kevin: use callback?
-        oldstatus = True
-        while True:
-            await asyncio.sleep_ms(500)
-            status = self.cl.status()
-            if status != oldstatus:
-                oldstatus = status
-                data = ['status', status]
-                line = ''.join((ujson.dumps(data), '\n'))
-                await self.swriter.awrite(line)
+    async def server_status(self, status):
```

Comment:
That is what I had in mind. Have you tested it?

Reply:
No I have not as I don't have any pyboard. It is implemented in the same way as my examples, so it should work but I'd have configure another esp to actually read the message that the coroutine sends using i2c.

Reply:
OK, no problem. It looks good, and I'll test when I merge.